packages/ecs-agent/ecs-agent.spec

```diff
@@ -83,7 +85,7 @@ go build -a \
   sed -i "s/~~digest~~/${DIGEST}/" image/config.json
   install -m 0644 %{S:7} image/manifest.json
   install -m 0644 %{S:8} image/repositories
-  %tar_cf ../../../amazon-ecs-pause.tar -C image .
+  %tar_cf ../../amazon-ecs-pause.tar -C image .
```

Comment:
How come this had to be moved?

Reply:
The Agent's container build process changed. A directory level was removed and restructured which changes this path (along with requiring us to build the executable explicitly now). 

See commit message for links to the relevant change: [`f48003f` (#1201)](https://github.com/bottlerocket-os/bottlerocket/pull/1201/commits/f48003ffa7c9cdf330f29fa3c23ac6c9775feb8c)